content/docs/user-guide/scripting/lua/debugging-tutorial.md

```diff
@@ -1,171 +1,144 @@
+    
+    {{< note >}}
 Connection is facilitated by GridHub, which is O3DE's central connection hub for debugging. GridHub starts automatically when Lua Editor is started and must be running in the background for Lua Editor to find targets it can connect to.
+{{< /note >}}
 
 1. In the Lua Editor toolbar, click **Target: None**, and then click **Editor(*ID*)** to connect to O3DE Editor.
-![Target selector](/images/user-guide/lua-editor-debugger-target-editor.png)
-**Note**
+
+    ![Target selector](/images/user-guide/scripting/lua/lua-editor-debugger-target-editor.png)
+    
+    {{< note >}}
 You may need to expand the Lua Editor window to see the buttons on the Lua Editor toolbar for the next few steps.
+{{< /note >}}
 
 1. In the Lua Editor toolbar, leave **Context** setting at **Default** for the debugging context. The default setting is good for debugging component entity scripts such as the one in this tutorial.
-![Context selector](/images/user-guide/lua-editor-debugger-context-choose.png)
+
+    ![Context selector](/images/user-guide/scripting/lua/lua-editor-debugger-context-choose.png)
 
 1. The **Debugging** icon turns green to show that Lua Editor and O3DE Editor are connected:
-![Lua Editor connected to O3DE Editor](/images/user-guide/lua-editor-debugger-connected-icon.png)
 
-   Click **Classes** in the **Class Reference** to show the available Lua libraries. You can do the same for **EBuses** and **Globals**.
-![Classes Reference](/images/user-guide/lua-editor-debugger-class-reference-pane.png)
-![Classes](/images/user-guide/lua-editor-debugger-class-reference-pane-open.png)
-**Note**
+    ![Lua Editor connected to O3DE Editor](/images/user-guide/scripting/lua/lua-editor-debugger-connected-icon.png)
+
+    Click **Classes** in the **Class Reference** to show the available Lua libraries. You can do the same for **EBuses** and **Globals**.
+    
+    ![Classes Reference](/images/user-guide/scripting/lua/lua-editor-debugger-class-reference-pane.png)
+    
+    ![Classes](/images/user-guide/scripting/lua/lua-editor-debugger-class-reference-pane-open.png)
+    
+    {{< note >}}
 The class reference feature is active only for the default context and component entity scripts.
+{{< /note >}}
 
-   After you connect, you can pause the execution of a given script by setting breakpoints.
+    After you connect, you can pause the execution of a given script by setting breakpoints.
 
-1. In the Lua Editor toolbar, click the **Breakpoints** icon ![Image NOT FOUND](/images/user-guide/lua-editor-debugger-breakpoints-icon.png) to show the **Breakpoints** window.
+1. In the Lua Editor toolbar, click the **Breakpoints** icon ![Breakpoints Icon](/images/user-guide/scripting/lua/lua-editor-debugger-breakpoints-icon.png) to show the **Breakpoints** window.
```

Comment:
No need to do this now, but now that we have svg icons in static/images/icons, we should look to get svg versions of these icon used on this page added to the directory and use the new icon shortcode.

Reply:
Ok, I'll make a pass to replace with svg icons.